spec/master_spec.js

```diff
@@ -0,0 +1,160 @@
+var noop = require("lodash.noop"),
+    cluster = require("cluster"),
+    runMasterJasmine = require("../lib/master");
+
+describe("master", function() {
+  beforeEach(function() {
+    var fakeRunJasmine = this.fakeRunJasmine = jasmine.createSpy("fakeRunJasmine");
+    spyOn(require("module"), "_load").and.callFake(function() {
```

Comment:
I don't think we should be spying on `module._load` to perform our tests. If something is a dependency that we need to be injected, it should be _explicitly_ injected through a constructor parameter or something.

Reply:
In the `../lib/master`, it will call `runJasmine` and I do not care that call will set `jasmine` object correctly. I only care about if it calls runJasmine. In this situation, how do I test?

Reply:
I specifically _want_ to force the injection of the fake Jasmine object if it is needed for testing. If this feels like it makes the arguments list too long, we can move towards accepting an `options` object as the only parameter, and pull the parts off from there.

I'm also going to "Unresolve" the later conversations for my own tracking purposes.